aspnetcore/mvc/views/tag-helpers/builtin/resources/CacheTagHelper.md

```diff
@@ -0,0 +1,278 @@
+﻿[Back To Built In Tag Helpers List](../../builtin.md)
+
+
+# CacheTagHelper
+
+By [Peter Kellner](http://peterkellner.net) 
+
+
+The `CacheTagHelper` provides the ability to dramatically improve the performance of your ASP.NET core app by caching its content to the internal ASP.NET core cache provider.
+
+A simple example that shows the `CacheTagHelper` in action sets the current time in the content area of the Cache Tag Helper. The Razor View Engine sets the default of `expires-after` to be 20 minutes (this is the default if you specify no addional attributes).
+
+Here is an example of the code you would include in your `cshtml` page to achieve the default cache behavior described above.
+
+```HTML
+<Cache>@DateTime.Now<Cache>
+```
+
+If you repeatedly refresh the page that contains the`Cache`tag above, the first time the current`DateTime`will be shown.  Then, repeated refreshes of the page will continue to show the first`DateTime` shown until the cache expires (which is likely 20 minutes in this example).
+
+You can achieve more control of the cache duration of the content by setting any of the following attributes.
+
+
+
+## The attributes are defined as follows
```

Comment:
As with the prior doc, I suggest a section title over a phrase ... `## Attributes`